.ibm/images/Dockerfile

```
@@ -64,12 +70,12 @@ RUN apt-get update -y && \
     apt-get install -y skopeo
 
 # Install umoci
-RUN curl -LO "https://github.com/opencontainers/umoci/releases/download/v0.4.7/umoci.amd64" && \ 
+RUN curl -LO "https://github.com/opencontainers/umoci/releases/download/v0.4.7/umoci.amd64" && \
     chmod +x umoci.amd64 && \
     mv umoci.amd64 /usr/local/bin/umoci
 
 # Install opm
-RUN curl -LO "https://github.com/operator-framework/operator-registry/releases/download/v1.23.0/linux-amd64-opm" && \ 
+RUN curl -LO "https://github.com/operator-framework/operator-registry/releases/download/v1.23.0/linux-amd64-opm" && \
```

Should this be newer? 

```suggestion
RUN curl -LO "https://github.com/operator-framework/operator-registry/releases/download/v1.51.0/linux-amd64-opm" && \
```